### DISCUSSION TITLE
Why "Verify Button" on Test email redirect to http://localhost:8090/_/#/auth/confirm-verification/....

### DISCUSSION THREAD

**Discussion by Vedsaga** [pocketbase/pocketbase#2879] [Q&A] [answered]
I have deployed pocketbase on AWS EC-2 instance behind the Nginx.

 And, on cloudflare I config the IP to point the for eg. `sub.domain.com` wards the public-ip.
 
Instead it should open `https://sub.domain.com/_/#/auth/confirm-verification/....` correct? Not sure what is that which I am doing wrong?

As, I can access the console `https://sub.domain.com/_/` just that for the email verification this is happening.

**Comment by ganigeorgiev** [comment, marked as answer]
You can change the default application url that is used in the mail templates from the _"Admin UI > Settings > Application > Application url"_ input.

**Reply by Vedsaga** [reply]
Thank you for quick response, what it should be exactly ? Simply `https://sub.domain.com/ ` or do I need to add any port etc?

**Reply by ganigeorgiev** [reply]
You should use the full url where your PocketBase instance is accessible (_in your case `https://sub.domain.com/` seems to be the correct one based on the above description_).

In general you can change the email template url to anything (not just the `{APP_URL}` placeholder) from the _"Admin UI > Settings > Mail settings > {:template} > Action url"._
